README.md

```diff
@@ -15,34 +15,41 @@ This repo consists of:
 
 ## Usage
 
-Start with a Python environment. Install `copier` and `poetry`:
+Start with a Python environment – we typically use [conda](https://docs.anaconda.com/anaconda/install/), but any environment manager will work. Install `copier` and `poetry`. Check that you have `python` installed or updated as well.:
 
 ```bash
+conda install conda-forge::poetry
```

Comment:
Installing poetry into the same environment as the rest of the dependencies (using conda or pip) has actually proven to cause some problems (it has some version pins that can conflict with dbt-related dependencies). Can we change these two lines to:

```bash
pipx install copier poetry
```
?

This would change again if we ever switch to uv, but for now I think that's the most reliable way to bootstrap a python environment